src/components/Code/Code.js

```diff
@@ -41,8 +41,57 @@ const Pre = styled.pre`
   font-size: ${props => props.theme.fontSizes.small};
 `;
 
+const PaddedLine = styled.div`
+  &:not(:last-child) {
+    padding-bottom: 8px;
+  }
+
+  /* required to make jsx children work with adding '\n' in multiLine */
+  & > div {
+    display: inline;
+  }
+`;
+
 const trimString = node => (isString(node) ? trim(node) : node);
 
+const formatSingleCommand = children =>
+  isFunction(children) ? children() : trimString(children);
+
+function formatMultiString(string) {
+  return trim(string)
+    .split('\n')
+    .map((line, i) => (
+      <PaddedLine key={i}>
+        {line}
+        {'\n'}
+      </PaddedLine>
+    ));
+}
+
+function formatMultiCommand(raw) {
+  let children = raw;
+
+  if (isFunction(children)) {
+    children = children();
+  }
+  const count = React.Children.count(children);
+
+  if (count === 1 && isString(children)) {
+    return formatMultiString(children);
+  }
+
+  if (count === 1) {
+    return children;
+  }
```

Comment:
Do we even need this case for what is supposed to be a `multiCommand`? If yes, maybe put it inside one `if (count === 1)` block?

Reply:
Fair point, more just a guard against having the prop on with a single child. Moved into single `if` block